src/components/WelcomeBack/WalletList.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+    <div class="content">
+      <p class="text">
+        Select which wallet do you want to unlock
+      </p>
+      <ul class="list">
+        <li v-for="wallet in wallets" :key="wallet.id">
+          <router-link :to="`/welcome-back/unlock/${wallet.id}`">
```

Comment:
if we add a router-link, we should also add a css class to remove text-decoration